sdk/data/azcosmos/cosmos_client.go

```diff
@@ -238,6 +238,26 @@ func (c *Client) sendDeleteRequest(
 	return c.executeAndEnsureSuccessResponse(req)
 }
 
+func (c *Client) sendBatchRequest(
+	path string,
+	ctx context.Context,
```

Comment:
For funcs/methods that take a `context.Context`, by convention, it should always be the first parameter.